File: moments.api/moments.space/moments.space.api/Models/Moment.cs
using System;
using MongoDB.Bson.Serialization.Attributes;

namespace moments.space.api.Models
{
    public class Moment
    {
        [BsonId]
        public int Id { get; set; }
        public string NasaId { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string Href { get; set; }
        public Category Category { get; set; }
        public string CreatedBy { get; set; }
        public DateTime CreationDate { get; set; }
    }
}

File: moments.ui/src/app/moment.ts

import { Category } from "./category";

export class Moment {
  id: number;
  nasaId: string;
  title: string;
  description: string;
  href: string;
  category: Category;
  createdBy: string;
  creationDate: Date;

  constructor() {
    this.title = '';
    this.description = '';
    this.href = '';
    this.category = new Category();
  }
}
